spec/features/course/assessment/question/programming_management_spec.rb

```diff
@@ -51,10 +51,66 @@
           expect(new_question.staff_only_comments).to include(attributes[:staff_only_comments])
           expect(new_question.question_assessments.first.skills).to contain_exactly(skill)
           expect(new_question.language).to eq(attributes[:language])
-          expect(new_question.template_files.first.content).to eq(template)
+          expect(new_question.template_files.first.content).to include(template)
         end
       end
 
+      scenario 'I can create a new question in an autograded assessment' do
+        skill = create(:course_assessment_skill, course: course)
+        assessment = create(:assessment, :autograded, course: course)
+        visit new_course_assessment_question_programming_path(course, assessment)
+
+        attributes = attributes_for(:course_assessment_question_programming)
+        template = "print('Hello World')"
+
+        fill_in 'Title', with: attributes[:title]
+        fill_in 'Maximum grade', with: attributes[:maximum_grade]
+
+        fill_in_react_ck 'textarea[name="question.description"]', attributes[:description]
+        fill_in_react_ck 'textarea[name="question.staffOnlyComments"]', attributes[:staff_only_comments]
+
+        find_field('Skills').click
+        find('li', text: skill.title).click
+
```

Comment:
Add a check for the warning text when the assessment is autograded, but the question is not to be evaluated and tested?